hypothesis/client
# Install wicked-good-xpath dependency from npm

wicked-good-xpath is a dependency that polyfills `document.evaluate` for IE 11. This function is used when anchoring range selectors. This PR removes the vendored copy in the source tree and installs it via npm instead.

This makes it easier to see what version we are using and get
automated notifications if the package is updated for any reason.

 - Update to current release of wicked-good-xpath from npm
 - Remove reference to IE 10 in comment since we now only support IE 11

----

To test manually, without IE 11:

1. Go to http://example.com
2. Open dev tools and run `delete Document.prototype.evaluate` in console. Evaluate `document.evaluate` in the console and it will print `undefined`
3. Load Hypothesis client into page using bookmarklet
4. Evaluate `document.evaluate` again in console. It should now refer to the function from the polyfill (the console may mention `wgxpath` in the location)
